python/gui/auto_generated/qgsmapcanvas.sip.in

```diff
@@ -1145,6 +1145,16 @@ Emitted when the map canvas temporal range changes.
 .. versionadded:: 3.14
 %End
 
+
+    void contextMenuPopulated( QMenu *menu, QgsMapMouseEvent *event );
```

Comment:
Don't forget to run the prepare commit script to update this. You'll probably need to run `scripts/sipify_all.sh` now in order to bulk update

Reply:
Arrgh. Sorry for that.